examples/thih/src/thih.lisp

```diff
@@ -54,7 +54,7 @@
     (TVar Tyvar)
     (TCon Tycon)
     (TAp Type Type)
-    (TGen Integer))
+    (TGen UFix))
```

Comment:
Leave this as integer. It matches the paper.

Reply:
I'm not sure I agree that a closer correspondence to the paper is more valuable here than the added conversions I'll have to add to `quantify`. I think using `UFix` is both more correct and cleaner here, and the only reason the paper uses `Integer` is because Haskell didn't have unsigned integer types until quite recently.

Reply:
I guess UFix is fine then